src/app/app.component.html

```diff
@@ -1,72 +1,58 @@
+<console [activePlanet]=activePlanet></console>
 
 <div id="circle-orbit-container">
-  <img id="constellation1" src="./assets/constellation1.png">
-  <img id="constellation2" src="./assets/constellation2.png">
-  <img id="constellation3" src="./assets/constellation3.png">
-  <img id="constellation4" src="./assets/constellation4.png">
-  <img id="constellation5" src="./assets/constellation5.png">
-  <img id="rocket" src="./assets/rocket.png">
-
-  <div id="sun-orbit">
-    <img  class="sun" src="./assets/sun.png">
+  <div id="pluto-orbit" class="orbit">
+    <img (click)="highlightElement('pluto')" class="planet pluto" src="./assets/pluto.png">
   </div>
-  <div id="mercury-orbit">
-    <img class="planet mercury" src="./assets/Mercury.png">
+  <div id="neptune-orbit" class="orbit">
+    <img (click)="highlightElement('neptune')" class="planet neptune" src="./assets/neptune.png">
```

Comment:
you don't have to have these setup in the attributes, you can add a class and add a handler to all o fhtme in js based on the class